xml/operations-maintenance-controller-full_recovery_test.xml

```diff
@@ -205,7 +205,10 @@ var/lib/ardana/service.osrc</screen>
      <title>Execute backup of Cassandra</title>
      <para>Execute backup of Cassandra</para>
      <para>Create cassandra-backup-extserver.sh script on all
-     controller nodes</para>
+     controller nodes where Cassandra runs, which can be determined
+     by running this command on deployer</para>
+<screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;ansible FND-MDB --list-hosts</screen>
```

Comment:
FND-CDB is where Cassandra runs... Apologies if I typo'd last time...
FND-MDB is where MariaDB/Galera runs...

Reply:
Didn't mean to "approve" last time around.